src/init.lua

```diff
@@ -1023,30 +1023,28 @@ end
 
 local saveLoop
 
-if RunService:IsStudio() then
-	game:BindToClose(function()
-		Store.ServiceDone = true
-		Keep.ServiceDone = true
+game:BindToClose(function()
```

Comment:
Seems I forgot to add back a "not" when testing. It was intended to not BindToClose in studio so it does not cause the servers to stay up. And, playerremoving seems enough for this. If could just add that would be great

Reply:
And sorry for pushing something, but resolve the small conflict. just one-line removal